runtime/runtime.go

```diff
@@ -32,19 +32,24 @@ var SupportedRuntimes []RuntimeType = []RuntimeType{
 }
 
 type RunConfig struct {
+	// general
```

Comment:
Similar comment as above - please, do not use inline comments in structs as it breaks the format.